crates/containerd-shim-wasm/src/container/path.rs

```diff
@@ -13,10 +9,10 @@ pub trait PathResolve {
     // Further contraints can be added by calling filtering the returned iterator.
     fn resolve_in_dirs<'a>(
```

Comment:
I think `'a` is not needed anymore, right?
```suggestion
    fn resolve_in_dirs(
```